docs/Configuration/fleet-server-configuration.md

```diff
@@ -269,6 +299,33 @@ Use a TLS connection to the Redis server.
     use_tls: true
   ```
 
+### redis_sts_assume_role_arn
+
+Amazon Resource Name (ARN) of the AWS Security Token Service (STS) role to use for Redis authentication. 
+
+If set, Fleet uses AWS Identity and Access Management (IAM) authentication instead of basic authentication set by `redis_username` and `redis_password` optional.
```

Comment:
```suggestion
If set, Fleet uses AWS Identity and Access Management (IAM) authentication instead of basic authentication set by `redis_username` and `redis_password` (optional).
```